tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -2963,17 +2963,10 @@ public function testCanonicalizedLocaleForMoment(?string $expected, string $orig
 
     public function testIsGrantedAffirmative(): void
     {
-        $this->securityChecker->method('isGranted')->willReturnCallback(static function (string $role): bool {
-            if ('foo' === $role) {
-                return false;
-            }
-
-            if ('bar' === $role) {
-                return true;
-            }
-
-            return false;
-        });
+       $this->securityChecker->method('isGranted')->willReturnMap([
+            ['foo', null, false],
+            ['bar', null, true],
+        ]);
 
         $this->assertTrue($this->twigExtension->isGrantedAffirmative(['foo', 'bar']));
```

Comment:
IMO, we should add some expectations for this method passing the arguments 2 and 3.